### PR TITLE
Ks 2022 05 issues

### PR DESCRIPTION
**Pull request by Rineee**

### CODE REVIEW

File: apps/userdashboard/serializers.py [hunk 1]
```diff
@@ -174,3 +176,29 @@ def get_num_active_notifications(self, comment):
             return self._get_num_pending_notifications(comment)
         else:
             return self._get_num_all_notifications(comment)
+
+    def update(self, instance, validated_data):
```
Comment (Rineee) [inline apps/userdashboard/serializers.py:180]: This is essentially the update method from rest_framework.serializers.ModelSerializer, I just added ignore_modified=True to the instance.save()

Comment (fuzzylogic2000) [reply apps/userdashboard/serializers.py:180]: Could you add a link to that method in a docstring? Not sure that is the best way to remember what we did, but might help.

**Review by fuzzylogic2000 — APPROVED** [review]
Nice! Just maybe add some more explanation for the first commit in the code?

File: apps/userdashboard/serializers.py (hunk 1 above)
Comment (Rineee) [reply apps/userdashboard/serializers.py:180]: Yes, good idea!

**Review by fuzzylogic2000 — APPROVED** [review]
Whoop whoop!

**Pull request merged**